under_development/tensorflow_iris_byom/tensorflow_BYOM_iris.ipynb

```diff
@@ -68,47 +68,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import os\n",
```

Comment:
Typos in markdown above... "mdoe" instead of "mode".  And "enstimator" should be "estimator".  Also instead of "Supply the role in the variable below", let's go with "If the get_execution_role() function does not return a role with the appropriate permissions, you'll need to specify an IAM role arn that does."